src/classes/LineItemTriggerHandler.cls

```diff
@@ -42,7 +46,7 @@ public class LineItemTriggerHandler {
 		for (Order__c o : orders) {
 			Decimal orderAmount = orderToAmountMap.get(o.Id); 
 			o.Purchased__c = (orderAmount!=null) ? orderAmount : 0;
-//			update o;
+			update o;
```

Comment:
This DML statement is inside a loop. Use a collection to store relevant data, and move the DML statement outside the loop.